gcc/rust/ChangeLog

```diff
@@ -3238,7 +3238,7 @@
 	* operator.h: New.
 
 
-Copyright (C) 2022-2023 Free Software Foundation, Inc.
+Copyright (C) 2022-2024 Free Software Foundation, Inc.
```

Comment:
Never touch ChangeLog file, these automatically modified by various scripts :)

Reply:
Sure 👍 
Are there any other changes you would recommend?